python/fixrgraph/extraction/run_extractor.py

```diff
@@ -329,19 +329,28 @@ def __init__(self, in_dir,
     def _call_sub(log, repo, args, cwd=None, wait=True):
         """Call a subprocess.
         """
-        logging.info("Executing %s" % " ".join(args))
-
-        # not pipe stdout - processes will hang
-        # Known limitation of Popen
-        proc = subprocess.Popen(args, cwd=cwd)
-        proc.wait()
+        write_log(log,repo,"Executing %s" % " ".join(args),"info")
+
+        if log is None:
+            #TODO: check if this is still the case
```

Comment:
@smover I piped the stdout and stderr into the logger so we can use them for the i/o discussed in the musedev meeting.  There is a note that piping stdout causes the process to hang.  I did not observe a hang when testing it.  Do you know what this comment may be about?  Can I remove it?